introduction.md

```diff
@@ -0,0 +1 @@
+# Hello, My name is Matt and I'm taking the codeschool course for Mastering Github.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.